Documentation/MRTK_PackageContents.md

```diff
@@ -1,25 +1,40 @@
 # Mixed Reality Toolkit packages
```

Comment:
Both of these Packages.md files seem to have a lot of overlap (indeed, both of them begin with the same title "Mixed Reality Toolkit packages".

Duplication is the bane of a lot of things - curious if you think it would be worth converging the two?

Reply:
i was thinking that earlier.... i'll fold them